### PR TITLE
add webhook for NatsJetstreamchannel

### PR DESCRIPTION
**Pull request by zhaojizhuang**
part 2 of  #145 
add webhook for NatsJetstreamchannel

**Release Note**

```release-note
add webhook for NatsJetstreamchannel
```

**Docs**

### CODE REVIEW

File: cmd/webhook/main.go [hunk 1]
```diff
@@ -31,13 +31,15 @@ import (
...
+	"knative.dev/eventing-natss/pkg/apis/messaging/v1alpha1"
 	"knative.dev/eventing-natss/pkg/apis/messaging/v1beta1"
 )
 
 var ourTypes = map[schema.GroupVersionKind]resourcesemantics.GenericCRD{
 	// For group messaging.knative.dev.
 	// v1beta1
-	v1beta1.SchemeGroupVersion.WithKind("NatssChannel"): &v1beta1.NatssChannel{},
+	v1beta1.SchemeGroupVersion.WithKind("NatssChannel"):          &v1beta1.NatssChannel{},
+	v1alpha1.SchemeGroupVersion.WithKind("NatsJetStreamChannel"): &v1alpha1.NatsJetStreamChannel{},
```
Comment (vaikas) [inline cmd/webhook/main.go:42]: nit: update comment on L40?

**Review by vaikas — COMMENTED** [review]
/lgtm
/approve

**Review by vaikas — COMMENTED** [review]
/lgtm
/approve

File: cmd/webhook/main.go (hunk 1 above)
Comment (zhaojizhuang) [reply cmd/webhook/main.go:42]: yeah, update when next PR

**Pull request merged**